odoo/addons/base/models/ir_ui_view.py

```diff
@@ -69,17 +69,12 @@ class ViewCustom(models.Model):
     _order = 'create_date desc'  # search(limit=1) should return the last customization
     _rec_name = 'user_id'
     _allow_sudo_commands = False
+    _user_id_ref_id = models.Index('(user_id, ref_id)')
```

Comment:
the definition after the fields ?

Reply:
Ok, it is sometimes done differently. Every time, it depends on where the `_sql_constraint` was.